utils/streams.js

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env node
+
+import { default as Argv } from 'yargs';
+import fs from 'fs';
+import { default as through } from 'through2';
+import csvjson from 'csvjson';
+import path from 'path';
+import request from 'request';
+
+const BASE_CSS_URL = 'https://www.epam.com/etc/clientlibs/foundation/main.min.fc69c13add6eae57cd247a91c7e26a15.css';
+
+const actions = {
+  io: inputOutput,
+  transform,
+  csvToJsonStream,
+  csvToJsonFile,
+  cssBundler
+};
+
+export default actions;
+
+if (require.main === module) {
+  cliRun();
+}
+
+function cliRun() {
+  const argv = Argv
+    .version('1.0.0')
+    .usage('Usage: $0 --action actionType [--file filePath]')
+    .example('$0 --action=io --file=./src/example.js')
+    .example('$0 --action transform --file=./data/test.txt')
+    .example('$0 -a csvToJsonStream -f ./data/test.csv')
+    .example('$0 -a csvToJsonFile -f ./data/test.csv')
+    .example('$0 --action cssBundler --path ./data/css')
+    .demandOption(['action'])
+    .option('action', {
+      alias: 'a',
+      describe: 'choose an action',
+      choices: ['io', 'transform', 'csvToJsonStream', 'csvToJsonFile', 'cssBundler']
+    })
+    .option('file', {
+      alias: 'f',
+      describe: 'provide a path to file (mandatory for -a=csvToJsonFile or -a=io)'
+    })
+    .option('path', {
+      alias: 'p',
+      describe: 'provide a path to folder to bundle CSS files (mandatory for --action=cssBundler)'
+    })
+    .help('h')
+    .argv;
+  
+  if (argv.action) {
+    if(argv.action === 'cssBundler') actions[argv.action](argv.path);
+    else actions[argv.action](argv.file);
+  }
+}
+
+/* stream helpers*/
+function inputOutput(filePath) { /* 4, babel-node ./utils/streams.js -a io -f ./data/test.txt */
+  if (fs.existsSync(filePath)) fs.createReadStream(filePath).pipe(process.stdout);
+  else console.log('Please specify correct file path');
+}
+
+function transform() { /* 5, node ./utils/streams.js -a transform */
```

Comment:
Quite difficult to follow the logic with your code style.
```js
transform() {
  stdInInformUser();
}

csvToJsonStream() {}
csvToJsonFile() {}
stdInInformUser() {}
```
Wtf? Why there are two functions between transforrm and stdInInformUser? And why it's placed below and not above? It's not reusable function. It would be better to group them
```js
stdInInformUser() {}
wirte() {}
end() {}
transform() {}
```